data/org.eclipse.birt.report.engine.script.javascript/src/org/eclipse/birt/report/engine/javascript/JavascriptVersion.java

```diff
@@ -84,6 +93,28 @@ public String getConfiguredECMAScriptVersion() {
 		return this.configuredECMAScriptVersion;
 	}
 
+	/**
+	 * Get the EMCAScript security in use
+	 *
+	 * @return the ECMAScript security in use
+	 */
+	public boolean isECMAScriptSecurityEnabled() {
+		return this.configuredEcmaScriptSecurity;
+	}
+
+	/**
+	 * Evaluate the system property to use the javascript security based on
+	 * certificates
+	 */
+	private void evaluateEcmaScriptSecurity() {
+		/* System property: -Dbirt.ecmascript.security */
+		String configuredEcmaScriptSecurityProperty = System.getProperty(ECMA_SCRIPT_SECURITY_PROPERTY_KEY);
+		if (configuredEcmaScriptSecurityProperty != null
+				&& configuredEcmaScriptSecurityProperty.equalsIgnoreCase(ECMA_SCRIPT_SECURITY_ENABLED)) {
+			this.configuredEcmaScriptSecurity = true;
+		}
+	}
+
```

Comment:
Same as earlier, but I would prefeer these to be static methods / fields, since there are no way to alter them per JavaScriptVersion -instance.

Reply:
Is changed completely.